dev.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Cool. I just realized your point about this file, and it might be a good reference resource for downstream integrations. We should also remember to add a bit of context to the README to highlight the critical parts of this file for quick reference.

Reply:
Just a small idea but perhaps we can rename this file to `example.html` to better communicate its purpose.

Reply:
I added some comments to the file and renamed it to `example.html`. This name is better as it's exactly what the file is for.